### PR TITLE
added metrics

### PR DESCRIPTION
**Pull request by allenywang**

### CODE REVIEW

File: annotator/static/player.js [hunk 1]
```diff
@@ -193,6 +199,9 @@ class Player {
     }
 
     drawAnnotationOnRect(annotation, rect) {
+        if (this.metrics.annotationsStartTime == null) {
```
Comment (antingshen) [inline annotator/static/player.js:202]: Use `===`

**Comment by antingshen** [comment]
Looks good to me.

**Comment by antingshen** [comment]
@szhu I just realized we're using the word "metrics" to mean something else now.. remember when I asked what "metrics" were before?

**Comment by szhu** [comment]
Yes. :/ I'll change the name back next time I touch that part of the code.

**Pull request merged**